p4-16/psa/PSA.mdk

```diff
@@ -293,15 +293,24 @@ few packets dropped for these reasons (perhaps only one).
 ## PSA type definitions {#sec-psa-type-definitions}
 
 Each PSA implementation will have specific bit widths for the
-following types in the data plane.  These widths should be defined in
-the target specific `psa.p4` include file.  They are expected to
-differ from one PSA implementation to another.
-
-For each one of these types, the P4 Runtime API uses values with the
-bit width of the corresponding `InHeader_t` type below to hold a value
-of that type, independent of the target.  All PSA implementations must
-use sizes for these types no wider than the corresponding types with
-`InHeader_t` in their names shown below.
+following types in the data plane.  These widths are defined in the
+target specific `psa.p4` include file.  They are expected to differ
+from one PSA implementation to another[^PSAP4TargetSpecific].
+
+[^PSAP4TargetSpecific]: It is expected that `psa.p4` include files for
+    different targets will typically be nearly identical to each
+    other.  Besides the possibility of differing bit widths for these
+    PSA types, the only expected differences between `psa.p4` files
+    for different targets would be annotations on externs, etc. that
+    the P4 compiler for that target needs to do its job.
+
+For each of these types, the P4 Runtime API[^P4RuntimeAPI] uses bit
+widths independent of the targets, large enough to hold a value for
+any target.  These widths are defined by the P4 Runtime API
+specification, but they are expected to be at least as large as the
+corresponding `InHeader_t` type below.  All PSA implementations must
+use data plane sizes for these types no wider than the corresponding
+types with `InHeader_t` in their names.
```

Comment:
It may be just me, but this reads a bit awkward. Suggestion:
```
For each of these types, the P4 Runtime API[^P4RuntimeAPI] uses bit
 widths independent of the targets.  These widths are defined by the P4 Runtime API
 specification, and they are expected to be at least as large as the
 corresponding `InHeader_t` type below, such that they hold a value for
 any target.  All PSA implementations must use data plane sizes for these
 types no wider than the corresponding `InHeader_t`-defined types.
```

Reply:
Your suggestion looks like a reasonable replacement to me.  Copying yours.